### ISSUE TITLE
[Automated] PRs inserted in VS build main-34105.100

### ISSUE THREAD

**Issue by allisonchou** [dotnet/roslyn#69818]
[View Complete Diff of Changes](https://github.com/dotnet/roslyn/compare/44555193fd1135b5d53a2099f76fec91e0d1ebde...4d8f34750e4b1af8452ef5ce520dcbd372b81510?w=1)
- [Update TFM in tests to net8.0 (69669)](https://github.com/dotnet/roslyn/pull/69669)
- [Update configs after 17.8p2 snap (69799)](https://github.com/dotnet/roslyn/pull/69799)
- [Remove EditorFeatures.Cocoa and VSMac IVTs (69765)](https://github.com/dotnet/roslyn/pull/69765)
- [Pack our language server in a separate package for each platform (69786)](https://github.com/dotnet/roslyn/pull/69786)
- [Switch PatternMatcher to use a TemporaryArray instead of ArrayBuilder. (69784)](https://github.com/dotnet/roslyn/pull/69784)
- [Update the severity combo boxes in EditorConfig UX to reflect the correct semantics (69778)](https://github.com/dotnet/roslyn/pull/69778)
- [Validate that only BoundConversion can have a field typed Conversion (69785)](https://github.com/dotnet/roslyn/pull/69785)
- [Improve interpolated string handler compilation error (69671)](https://github.com/dotnet/roslyn/pull/69671)
- [Fix the ArgumentOutOfRangeException in inline rename (69708)](https://github.com/dotnet/roslyn/pull/69708)
- [Add extra validation for Compilations having the expected trees (69747)](https://github.com/dotnet/roslyn/pull/69747)
- [Add System.IO.Pipelines to Version.Details for source-build (69719)](https://github.com/dotnet/roslyn/pull/69719)
- [Bump telemetry for fix on ExeName/Version (69771)](https://github.com/dotnet/roslyn/pull/69771)
- [Upgrade MSBuildLocator to 1.6.1 (69769)](https://github.com/dotnet/roslyn/pull/69769)
- [Report missing diagnostic for typeless expression in binary operation (69727)](https://github.com/dotnet/roslyn/pull/69727)
- [Collection expression conversions: check elements are convertible (69627)](https://github.com/dotnet/roslyn/pull/69627)
- [Bail out for primary constructor parameters in IDE0059 analyzer (69748)](https://github.com/dotnet/roslyn/pull/69748)
- [Use raw strings in EditorFeatures tests (69729)](https://github.com/dotnet/roslyn/pull/69729)
- [Add comment and bug link for static graph restore false (69759)](https://github.com/dotnet/roslyn/pull/69759)
- [Work around restore issue (69710)](https://github.com/dotnet/roslyn/pull/69710)
- [Followup PR to https://github.com/dotnet/roslyn/pull/69677 to reduce allocations further. (69746)](https://github.com/dotnet/roslyn/pull/69746)
- [Pool lexed token arrays in SyntaxParser. (69499)](https://github.com/dotnet/roslyn/pull/69499)
- [Update PublishData.json (69752)](https://github.com/dotnet/roslyn/pull/69752)
- [fixing error formatting for generic attribute missing errors (69735)](https://github.com/dotnet/roslyn/pull/69735)
- [Bind fully-typed lambda only once for error recovery (69695)](https://github.com/dotnet/roslyn/pull/69695)
- [Pass project file name when building before running tests (69743)](https://github.com/dotnet/roslyn/pull/69743)
- [Enable the test method finder to find tests in nested classes. (69716)](https://github.com/dotnet/roslyn/pull/69716)
- [Update configs for 17.8 P3 (69741)](https://github.com/dotnet/roslyn/pull/69741)
- [Add restricted IVT to Intellicode (69742)](https://github.com/dotnet/roslyn/pull/69742)
